### PR TITLE
Created ox_inventory compatible version

### PR DESCRIPTION
**Pull request by Fritoqc**
Adds item description
Also updated to the last recent release.

image

### CODE REVIEW

File: client/nui.lua [hunk 1]
```diff
@@ -41,7 +41,7 @@ function NUI:SetupJobs()
     for key = 1, #Config.jobs, 1 do
         jobs[#jobs + 1] = {
             name = Config.jobs[key].label,
-            salary = QBCore.Shared.Jobs[Config.jobs[key].job].grades["0"].payment, -- Pulls the salary data from QBCore, Credit to 42MARTIN42 for the change, instead of using a set config value using - Config.jobs[key].salary,
```
Comment (NietThijmen) [inline client/nui.lua:44]: Why would a PR about ox_inventory change how the salary works?

File: config.lua [hunk 2]
```diff
@@ -2,9 +2,6 @@ QBCore = exports['qb-core']:GetCoreObject()
 
 Config = {}
 
-Config.RenewedPhone = false --> Set to true if you are using Renewed scripts their qb-phone
```
Comment (NietThijmen) [inline config.lua:5]: Dropping support for renewed phone isn't on our todo list

File: config.lua [hunk 3]
```diff
@@ -2,9 +2,6 @@ QBCore = exports['qb-core']:GetCoreObject()
...
-Config.idcard = "default" -- default, bl_idcard, qbx_idcard, um_idcard
```
Comment (NietThijmen) [inline config.lua:6]: We supported multiple idcards, why take it away :(

File: fxmanifest.lua [hunk 4]
```diff
@@ -1,8 +1,8 @@
 fx_version 'cerulean'
 games { 'rdr3', 'gta5' }
 
-author 'More Than Code'
-description 'Cityhall (mtc-cityhall) 1.0.0'
+author 'More Then Code'
```
Comment (NietThijmen) [inline fxmanifest.lua:4]: I guess we'll just change our name then.

File: readme.md [hunk 5]
```diff
@@ -34,7 +32,3 @@
 ![jobs](./showcase/jobs.png)
 ![items](./showcase/items.png)
 ![info](./showcase/info.png)
-
-## 👉 Join our community
```
Comment (NietThijmen) [inline readme.md:38]: Yeah, fuck people joining our discord!

File: server/server.lua [hunk 6]
```diff
@@ -25,10 +22,32 @@ RegisterNetEvent('mtc-cityhall:server:BuyIdentity', function(id)
         return 
     end
 
-    if not giveIdCard(src, item) then  -- Handles id_card items using qb-inventory or um_idcard if it was not an id_card give the item to the user
-        Player.Functions.AddItem(item.item, 1, false)
+    local info = {}
```
Comment (NietThijmen) [inline server/server.lua:25]: Please add this to server/idcard.lua

**Review by NietThijmen — CHANGES_REQUESTED** [review]
Just a few passive aggresive changes.

**Comment by FritoQC** [comment]
I didnt change that, the release version wasnt the same as the main branch.
Only file changed was server.lua, Added the changed files since it was the latest release

**Comment by NietThijmen** [comment]
> I didnt change that, the release version wasnt the same as the main branch.
> 
> Only file changed was server.lua, Added the changed files since it was the latest release

Yeah but why use the release instead of the main branch? As this script doesn't need releases as it's not build or nothing.

**Pull request closed without merging**